From 0aad0b40cac4aa631226208d5d39b93caa099255 Mon Sep 17 00:00:00 2001
From: Bryan Evenson <bevenson@melinkcorp.com>
Date: Fri, 22 Nov 2013 10:26:07 -0500
Subject: [PATCH 1/2] AT91: rtc: Enable RTC in device tree

Enables the RTC in the device tree for the AT91SAM9x5 family.

Signed-off-by: Bryan Evenson <bevenson@melinkcorp.com>
---
 arch/arm/boot/dts/at91sam9x5ek.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/at91sam9x5ek.dtsi b/arch/arm/boot/dts/at91sam9x5ek.dtsi
index 5d94f42..0e7de53 100644
--- a/arch/arm/boot/dts/at91sam9x5ek.dtsi
+++ b/arch/arm/boot/dts/at91sam9x5ek.dtsi
@@ -97,6 +97,10 @@
 			watchdog@fffffe40 {
 				status = "okay";
 			};
+
+			rtc@fffffeb0 {
+				status = "okay";
+			};
 		};
 
 		usb0: ohci@00600000 {
-- 
1.7.9.5

